plugins/vendor/zoom/lib/model/kZoomEngine.php

```diff
@@ -353,11 +353,18 @@ protected function createEntryDescriptionFromMeeting($meeting)
 	 * @param entry $entry
 	 * @throws kCoreException
 	 */
-	protected function setEntryCategory($zoomIntegration, $entry)
+	protected function createCategoryEntry($zoomIntegration, $entry)
```

Comment:
This code should move to categoryentry core object
Adding new funino categoryEntry::add(entry, category)

When it is added , you must behave the same way cateogryEntryService::add handles the moderation.